src/main/target/OMNIBUSF4/target.h

```diff
@@ -65,8 +60,6 @@
 #define INVERTER_PIN_UART3      PC9 // Omnibus F4 Pro Corners
 #elif defined(EXUAVF4PRO)
 #define INVERTER_PIN_UART6      PC8
-#else
-#define INVERTER_PIN_UART1      PC0 // PC0 used as inverter select GPIO XXX this is not used --- remove it at the next major release
```

Comment:
This should live for DYSF4PRO as it has controllable inverter on RX1 controlled by PC0 . Only the comment should be deleted.